R/entries.R

```diff
@@ -0,0 +1,12 @@
+#' @export
+entries <- function(df){
```

Comment:
I would rename to `hail_entries()`, I would also mention in the README that this function selects and explodes the data frame using `sparklyr.nested` and a link to the implementation. I think that would help users modify this if needed.